TenureInformationApi.Tests/V1/E2ETests/Fixtures/TenureFixture.cs

```diff
@@ -63,10 +71,26 @@ public void GivenNewTenureRequestWithValidationErrors()
                                         .With(x => x.SubletEndDate, DateTime.UtcNow)
                                         .With(x => x.EvictionDate, DateTime.UtcNow)
                                         .Create();
+            CreateSnsTopic();
 
             CreateTenureRequestObject = tenureRequest;
         }
 
+        private void CreateSnsTopic()
+        {
+            var snsAttrs = new Dictionary<string, string>();
+            snsAttrs.Add("fifo_topic", "true");
+            snsAttrs.Add("content_based_deduplication", "true");
+
+            var response = _amazonSimpleNotificationService.CreateTopicAsync(new CreateTopicRequest
+            {
+                Name = "tenurecreated",
+                Attributes = snsAttrs
+            }).Result;
+
+            Environment.SetEnvironmentVariable("NEW_TENURE_SNS_ARN", response.TopicArn);
+        }
```

Comment:
You don't need this here.
The test story class already has the `Aws collection` set on it meaning that the implementation within the `AwsIntegrationTests` class will have already been run.

Reply:
But the method in integration tests is currently private, also we want to create the SNS topic when the request object is created